src/python/gudhi/sktda/kernel_methods.py

```diff
@@ -0,0 +1,202 @@
+"""
+@author: Mathieu Carriere
+All rights reserved
```

Comment:
According to [wikipedia](https://en.wikipedia.org/wiki/All_rights_reserved), this is useless.